### PR TITLE
Add more unit tests for hyperdisk feature

### PR DESCRIPTION
**Pull request by sunnylovestiramisu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Add more tests to clarify use case, clean up test format.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/common/utils_test.go [hunk 1]
```diff
@@ -706,70 +706,82 @@ func TestConvertMiBStringToInt64(t *testing.T) {
 		expectError bool
 	}{
 		{
-			"valid number string",
-			"10000",
-			1,
-			false,
+			desc:        "valid number string",
+			inputStr:    "10000",
```
Comment (leiyiz) [inline pkg/common/utils_test.go:710]: why is 10000 converted to 1 and 1000Ki is also 1?

**Comment by leiyiz** [comment]
also consider change the name from `ConvertMiBStringToInt64(str string)` to `ConvertMiStringToInt64(str string)`?

File: pkg/common/utils_test.go (hunk 1 above)
Comment (sunnylovestiramisu) [reply pkg/common/utils_test.go:710]: The round up is doing roundedUp := volumeSizeBytes / allocationUnitBytes(Here we use MiB which is int 1048576)

10000 / 1048576 round up is 1
1000Ki / 1048576 round up is 1
1000k / 1048576 round up is 1 as well.

**Comment by sunnylovestiramisu** [comment]
> also consider change the name from `ConvertMiBStringToInt64(str string)` to `ConvertMiStringToInt64(str string)`?

That is a good suggestion, will change to that now

**Review by leiyiz — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**